### PR TITLE
mgmt, local generation for AppPlatform Spring - 2023-12-01

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to [issue#4883](https://github.com/Azure/sdk-release-request/issues/4883)

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by v-hongli1** [comment]
Fixed ci analyze exception, the detail info following as: 
https://dev.azure.com/azure-sdk/public/_build/results?buildId=3425198&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=68a849d6-f425-5f38-d450-7e2c3d1df885

**Comment by v-hongli1** [comment]
Fixed ci analyze exception, the detail infomation following as:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=3425198&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9

**Comment by XiaofeiCao** [comment]
> Fixed ci analyze exception, the detail infomation following as: https://dev.azure.com/azure-sdk/public/_build/results?buildId=3425198&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9

Why did service make `kPackVersion` readonly for `BuildServiceProperties`?

**Comment by weidongxu-microsoft** [comment]
DO NOT DO THIS UTILL THE PR IS REVIEWED https://github.com/Azure/azure-sdk-for-java/pull/38403/commits/13a561a4c61122c27f2012f18bb6c4e69e12e164

**Comment by XiaofeiCao** [comment]
@v-hongli1 Service confirmed it's readonly from the very beginning. 
Screenshot 2024-01-26 at 10 41 20
We can take the breaking change and add revapi.skip.
Please also mention it in changelog.md's breaking change section, thanks.

**Comment by v-hongli1** [comment]
`<revapi.skip>true</revapi.skip>` has been added into `pom.xml`

**Comment by weidongxu-microsoft** [comment]
changelog

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager-appplatform/CHANGELOG.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
 
 ## 2.35.0-beta.1 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
+- Removed `withKPackVersion` from `BuildServiceProperties`.
```
Comment (XiaofeiCao) [inline sdk/resourcemanager/azure-resourcemanager-appplatform/CHANGELOG.md:7]:
```suggestion
- Removed `withKPackVersion` from `BuildServiceProperties` as it's never functional in the backend.
```

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-appplatform/CHANGELOG.md]: Fixed in the new versiom.

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**